config.example.storage.wikimedia.yaml

```diff
@@ -0,0 +1,112 @@
+# RESTBase wikimedia example config
+
+# Load some project templates. These are referenced / shared between domains
+# in the root_spec further down.
+default_project: &default_project
+  x-modules:
+    - spec:
+        paths:
+          /{api:v1}:
+            x-modules:
+              - spec:
+                  paths:
+                    /key_value:
+                      x-modules:
+                        - path: sys/key_value.js
+                    /page_revisions:
+                      # We need to forward cookie to the API
+                      x-route-filters:
+                        - path: ./lib/mediawiki_auth_filter.js
+                      x-modules:
+                        - path: sys/page_revisions.js
+          /{api:sys}:
+            x-modules:
+              - spec:
+                  paths:
+                    /table:
+                      x-modules:
+                        - path: sys/table.js
+                          options:
+                            conf:
+                              backend: '{env(RB_TEST_BACKEND, sqlite)}'
+                              hosts: [localhost]
+                              keyspace: system
+                              username: cassandra
+                              password: cassandra
+                              defaultConsistency: one # or 'localQuorum' for production
+                              storage_groups:
+                                - name: default.group.local
+                                  domains: /./
+                              dbname: test.db.sqlite3 # ignored in cassandra, but useful in SQLite testing
+                    /action:
+                      x-modules:
+                        - path: sys/action.js
+                          options:
+                            apiUriTemplate: "{{'https://{domain}/w/api.php'}}"
+                            baseUriTemplate: "{{'https://{domain}/api/rest_v1'}}"
+
+
+# The root of the spec tree. Domains tend to share specs by referencing them
+# using YAML references.
+spec_root: &spec_root
+  title: "The RESTBase Storage Service"
+  x-sub-request-filters:
+    - type: default
+      name: http
+      options:
+        allow:
+          - pattern: /^https?:\/\/[a-zA-Z0-9\.]+\/w\/api\.php/
+            forward_headers: true
+  paths:
+    /{domain}: *default_project
+    # A robots.txt to make sure that the content isn't indexed.
+    /robots.txt:
```

Comment:
`robots.txt` will be matched by `/{domain}` above, so it needs to be listed before.

Reply:
Actually, it needs to be removed entirely. This is not going to be exposed outside.

Reply:
Let's not do that. Let's minimise the changes we make outside the split.

Reply:
mmm? This is a completely new config, so I don't get how not removing this would minimize any of the changes. It's a copy-paste artifact.

Reply:
No, it's not. It's the front-end component of the current RB. And the current status quo is that RESTBase-the-router exposes it. So let's keep it for now.

Reply:
This is not the frontend component :) This is all complicated, but this is the example of how the backend component will look like when we finish up with the transition :)

Reply:
/me facepalms. Dang you are right. And it says nicely in the file name lol. Ok, yes, let's remove it.